taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class LicenceValidateMixin:
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]  # noqa

```

Comment:
you really need #noqa?

Reply:
![image](https://user-images.githubusercontent.com/72568844/200882708-7fc07fcc-5d10-4efe-9f98-e2d0b54c1bff.png)